scrapyjs/middleware.py

```diff
@@ -43,24 +52,26 @@ def from_crawler(cls, crawler):
         return cls(crawler, splash_base_url, slot_policy)
 
     def process_request(self, request, spider):
-        splash_options = request.meta.get('splash')
+        splash_options = self.get_splash_options(request, spider)
         if not splash_options:
             return
 
+        elif request.meta.get("_splash_processed"):
+            return
```

Comment:
got it. 

note - I think it would be a bit safer to use `if` here, not `elif`, as both statements (https://github.com/scrapinghub/scrapyjs/pull/15#discussion-diff-28138514R56 and https://github.com/scrapinghub/scrapyjs/pull/15#discussion-diff-28138514R59) leading to `return` - it wouldn't change logic but who knows how this code will change in future